### ISSUE TITLE
firstVoteColumnIndex and firstVoteRowIndex issues for JSON CVRs

### ISSUE THREAD

**Issue by HEdingfield** [BrightSpots/rcv#325]
The GUI allows you to add a CVR to the list without first vote row and first vote column if the filename contains `.json`. However, it doesn't run this check again if you double click the fields in the list and modify them, thus forcing you to keep a value in these fields.

Also encountering this when I try to run validation:
```
2019-07-04 09:35:52 PDT SEVERE: Cast vote record file not found: C:\Users\Me\IdeaProjects\rcv\test.json
2019-07-04 09:35:52 PDT SEVERE: firstVoteColumnIndex is required: C:\Users\Me\IdeaProjects\rcv\test.json
2019-07-04 09:35:52 PDT SEVERE: firstVoteRowIndex is required: C:\Users\Me\IdeaProjects\rcv\test.json
```
Not sure if these latter two notes about the indices being required would disappear if it were actually able to load the .json file? If not, we may have another problem here.

**Comment by HEdingfield** [comment]
May be mitigated partially by PR #328.

**Comment by tarheel** [comment]
> Not sure if these latter two notes about the indices being required would disappear if it were actually able to load the .json file?

It won't require those fields if source = "CDF".

**Comment by HEdingfield** [comment]
Fixed with #359.
